docs/usage/recreate.rst.inc

```diff
@@ -36,10 +36,10 @@ Exclusion options
         | read exclude patterns from EXCLUDEFILE, one per line
     ``--exclude-caches``
         | exclude directories that contain a CACHEDIR.TAG file (http://www.brynosaurus.com/cachedir/spec.html)
-    ``--exclude-if-present FILENAME``
-        | exclude directories that contain the specified file
-    ``--keep-tag-files``
-        | keep tag files of excluded caches/directories
+    ``--exclude-if-present PATH``
+        | exclude directories that contain the specified file/directory
+    ``--keep-exclude-tags``
+        | keep tag files/directories (i.e.: arguments to --exclude-if-present) in otherwise excluded caches/directories
 
 Archive options
     ``--target TARGET``
```

Comment:
maybe do not update the .rst.inc file at all, these files are usually created / updated at release time.

changes in here also often cause merge conflicts e.g. when merging one branch into another.